src/commonmark/en/content/user/managing-dashboards.md

```diff
@@ -112,22 +112,32 @@ items (like an empty row), you can add spacer items to the dashboard.
 They are only visible in edit/create mode. In view mode, they are not
 displayed, but take up the defined space.
 
-**Spacer in edit/create mode:**
+Spacer in **edit/create mode**:
 
 ![](resources/images/dashboard/dashboard-spacer-edit-mode.png)
 
-**Spacer in view mode:**
+Spacer in **view mode**:
 
 ![](resources/images/dashboard/dashboard-spacer-view-mode.png)
 
-## Removing items
+### Removing items
 
 Remove items by clicking on the red trash can at the upper right of the
 item. Be aware that because of the "anti-gravity" setting in the
 dashboard, when you remove an item, the items that are positioned below
 the removed item will "rise" upwards.
 
-## Saving the dashboard
+### Print preview
+
+Click on the **Print preview** button to view how the dashboard would look in dashboard layout print.
+
+![](resources/images/dashboard/dashboard-edit-print-preview.png)
+
+Click on **Exit Print preview** to return to editing the dashboard.
+
+Note that some items may be moved down in order to avoid page breaks Items may also be shortened to fit on one page. Items that are shortened show an info icon in the upper right corner in preview. This icon is removed in the actual print.
```

Comment:
```suggestion
Note that some items may be moved down in order to avoid page breaks. Items may also be shortened to fit on one page. Items that are shortened show an info icon in the upper right corner in preview. This icon is removed in the actual print.
```